stored_requests/caches/nil_cache/nil_cache.go

```diff
@@ -8,13 +8,13 @@ import (
 // NilCache is a no-op cache which does nothing useful.
 type NilCache struct{}
 
-func (c *NilCache) Get(ctx context.Context, requestIDs []string, impIDs []string) (map[string]json.RawMessage, map[string]json.RawMessage) {
-	return nil, nil
+func (c *NilCache) Get(ctx context.Context, ids []string) map[string]json.RawMessage {
```

Comment:
Not a problem you introduced, but I have noticed something here. If the actual cache finds no hits, it returns an existing, but empty map. The nil cache returns a nil map. There is a subtle difference in at least the fact that you can expand an empty map, but you cannot assign any entries to a nil map. Most likely not an issue, but we may want the nil cache to act more like the actual cache.

Reply:
Given this comment https://github.com/prebid/prebid-server/blob/fb386190f4491648bb1e8d1b0345a333be1c0393/stored_requests/fetcher.go#L63 it looks like the caller expects a writeable map so I'll change to return `make(map[string]json.RawMessage)`